src/cms/templates/pages/page_tree_node.html

```diff
@@ -4,11 +4,13 @@
 {% load tree_filters %}
 <tr data-drop-id="{{ node.id }}" data-drop-position="left" class="drop drop-between h-3 -m-3 hidden level-{{page.depth}}"><td colspan="9"><div><span></span></div></td></tr>
 <tr data-drop-id="{{ node.id }}" data-drop-position="last-child" class="drop drop-on border-t border-b border-solid border-gray-200 hover:bg-gray-100{% if page.depth > 0 %} child level-{{page.depth}}{% endif %}">
-    <td class="single_icon">
-        <span data-drag-id="{{ page.id }}" data-node-descendants="{{ page|get_descendants }}" class="drag text-gray-800 block py-3 pl-4 pr-2 cursor-move" draggable="true">
-            <i data-feather="move"></i>
-        </span>
-    </td>
+    {% if enable_drag_and_drop %}
+        <td class="single_icon">
+            <span data-drag-id="{{ page.id }}" data-node-descendants="{{ page|get_descendants }}" class="drag text-gray-800 block py-3 pl-4 pr-2 cursor-move" draggable="true">
+                <i data-feather="move"></i>
+            </span>
+        </td>
+    {% endif %}
```

Comment:
```suggestion
    <td class="single_icon">
        <span data-drag-id="{{ page.id }}"
              data-node-descendants="{{ page|get_descendants }}"
              class="{% if enable_drag_and_drop %}drag cursor-move{% else %}cursor-not-allowed{% endif %} text-gray-800 block py-3 pl-4 pr-2"
              {% if enable_drag_and_drop %}
              draggable="true"
              {% else %}
              title="{% trans 'Drag & drop is disabled when filters are applied.' %}"
              {% endif %}>
            <i data-feather="move"></i>
        </span>
    </td>
    {% endif %}
```
I would propose to show the hierarchy even if drag & drop is disabled. Also, the checkboxes look a but buggy if they're not aligned...